src/card/aggregate-status/aggregate-status-card.component.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @ngdoc directive
- * @name patternfly.card.component:pfAggregateStatusCard
+ * @name patternfly.card.directive:pfAggregateStatusCard
```

Comment:
Why the change to `directive`?  pfAggregateStatusCard does appear to be a component:
```
angular.module( 'patternfly.card' ).component('pfAggregateStatusCard', {
```

Reply:
Needed for grunt-ui-docs to even show the component on the generated doc webpage.

Reply:
Yeah - something changed after angular 1.6 that requires the name match directive now (and components aren't natively supported by grunt-ui-docs or ngdocs)

Reply:
Ok, thought it was something along those lines